src/stactools/cli/commands/lint.py

```diff
@@ -0,0 +1,20 @@
+import click
+from stac_check.cli import cli_message as lint_message  # type: ignore
+from stac_check.lint import Linter  # type: ignore
+
+def create_lint_command(cli: click.Group) -> click.Command:
+
+    @cli.command("lint", short_help="Lint a stac object with stac-check.")
+    @click.argument("href")
+    def lint_command(href: str) -> None:
+        """Validates a STAC object.
+
+        Prints any validation errors to stdout.
+        """
+        linter = Linter(href, assets=True, links=True, recursive=False)
```

Comment:
Recursive right now only lints the first object and then uses pystac validation to check all of the children (without linting them). When it is improved it may be more useful.